sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -812,6 +813,36 @@ def progress_metrics(self):
           current_element_progress.fraction_remaining)
     return metrics
 
+  def monitoring_infos(self, transform_id):
```

Comment:
a bit surprised there is no testing file corresponding to operations.py. 

Maybe I missed something?

Reply:
Your not wrong, there is no operations_test.py.

A bunch of the testing in the Python SDK relies on pipeline level tests which test execution e2e.